tendrl/gluster_integration/manager/manager.py

```diff
@@ -73,47 +64,25 @@ def _run(self):
         LOG.info("%s complete" % self.__class__.__name__)
 
 
-class Manager(object):
-    """Manage a collection of ClusterMonitors.
-
-    Subscribe to ceph/cluster events, and create a ClusterMonitor
-
-    for any FSID we haven't seen before.
-
-    """
-
+class GlusterIntegrationManager(Manager):
     def __init__(self, cluster_id):
         self._complete = gevent.event.Event()
-
-        self._user_request_thread = EtcdThread(self)
-        self._discovery_thread = TopLevelEvents(self, cluster_id)
```

Comment:
The discovery thread is now initialized in base class and test_manager is updated accordingly.